serviceAccount/README.md

```diff
@@ -0,0 +1,90 @@
+# Change AD FS Service Account
+
+## Overview
+
+This powershell module allows the AD FS service account to be changed. Such functionality may be especially useful if the current service account has been compromised.
+
+The module exports four functions which may be used in conjunction to successfully migrate the service to a new account or recover from an error encountered along the way:
+
+1.Add-AdfsServiceAccountRule
+
+2.Remove-AdfsServiceAccountRule
+
+3.Update-AdfsServiceAccount
+
+4.Restore-AdfsSettingsFromBackup
+
+##
+The module exposes the following cmdlets:
+1. __```Add-AdfsServiceAccountRule```__ - Adds permission rule for the specified service account. Must be run prior to changing the service account on Windows Server 2016 or later.
+2. __```Remove-AdfsServiceAccountRule```__ - Removes permission rule for the specified service account. This should not be run until the AD FS service is verfifed to work with the new service account.
+3. __```Update-AdfsServiceAccount```__ - Changes the AD FS service account on the local machine. This cmdlet should be run on all secondary servers prior to execution on the primary machine.
+4. __```Restore-AdfsSettingsFromBackup```__ - Restores the AD FS service settings with a backup generated during either Add-AdfsServiceAccountRule or Remove-AdfsServiceAccountRule. This cmdlet can be used to recover if an error occurs during either of the afforementioned commands.
+## Requirements 
+
+1.The module is applicable for any AD FS farm and works for both SQL and WID environments.
```

Comment:
In order for the GitHub markdown to work for the numbers, there has to be a space between the period after the number, and the first letter of your sentence